src/ducks/modules/__tests__/sessions.test.js

```diff
@@ -43,6 +44,7 @@ describe('sessions reducer', () => {
         path: 'path/to/session',
         network: { ego: {}, nodes: [], edges: [] },
         promptIndex: 0,
+        updatedAt: 1487076708000,
```

Comment:
This assertion could be reverted to `toEqual` if you want that specificity.

(I switched to `MatchObject` and ignored timestamps, thinking at some point we'd add assertions that timestamps are modified under expected conditions.)

The other ones below would match entirely as well except for the network nodes.

Reply:
Ah, I can switch that back to ignoring the timestamps if you prefer. I'm not particular about it.

Reply:
I think its worth changing, and perhaps ignoring timestamps.